Runtime/PostProcessing/Shaders/CRT.shader

```diff
@@ -139,6 +150,91 @@ Shader "Hidden/HauntedPS1/CRT"
         return color;
     }
 
+    float3 QuadratureAmplitudeModulation(float3 colorYIQ, float2 positionFramebufferNDC)
+    {
+        float Y = colorYIQ.r;
```

Comment:
Not required, but by convention, I find it nice to only use .rgb when actually dealing with RGB values. In this case we YIQ is a luminance chroma space, so just to highlight this I like to use .xyz instead